test/common/testinfra.py

```diff
@@ -581,6 +581,10 @@ def update_status(revision, context, last, changes):
                 status = statuses.get(context, None)
                 baseline = BASELINE_PRIORITY
 
+                # modify the baseline slightly to favor older pull requests, so that we don't
```

Comment:
> Wouldn't that have the same effect?

Maybe, but that would rely on a non-documented observation that the list tends to be in that order right now. We don't specify any [sorting criteria](https://developer.github.com/v3/pulls/#parameters) when getting the list of pull requests right now. Even if we did use the age criterion, we would again have the problem with the last updated timestamp being counter-intuitive as a test runner prioritization criterion.